app/lib/import_export.rb

```diff
@@ -639,48 +375,32 @@ def extract_incomplete_patient_details(patient, fields)
     patient_details
   end
 
-  # Gets a hash of the latest transfers of each patient
-  def get_patients_transfers(patients)
-    transfers = Transfer.latest_transfers(patients)
-    jurisdictions = Jurisdiction.find(transfers.pluck(:from_jurisdiction_id, :to_jurisdiction_id).flatten.uniq)
-    jurisdiction_paths = Hash[jurisdictions.pluck(:id, :path).map { |id, path| [id, path] }]
-    Hash[transfers.pluck(:patient_id, :created_at, :from_jurisdiction_id, :to_jurisdiction_id)
-                  .map do |patient_id, created_at, from_jurisdiction_id, to_jurisdiction_id|
-                    [patient_id, {
-                      transferred_at: created_at.rfc2822,
-                      transferred_from: jurisdiction_paths[from_jurisdiction_id],
-                      transferred_to: jurisdiction_paths[to_jurisdiction_id]
-                    }]
-                  end
-        ]
-  end
-
-  # Gets a hash of the 2 latest laboratories of each patient
-  def get_patients_laboratories(patients)
-    Laboratory.where(patient_id: patients.pluck(:id)).order(specimen_collection: :desc).group_by(&:patient_id).transform_values { |v| v.take(2) }
-  end
-
   # Extracts assessment data values given relevant fields
   def extract_assessments_details(patients_identifiers, assessments, fields)
     if fields.include?(:symptoms)
-      conditions = ReportedCondition.where(assessment_id: assessments.pluck(:id))
-      symptoms = Symptom.where(condition_id: conditions&.pluck(:id)).order(:label)
-
-      conditions_hash = Hash[conditions&.pluck(:id, :assessment_id)&.map { |id, assessment_id| [id, assessment_id] }]
+      conditions_hash = Hash[assessments.joins(:reported_condition).pluck('conditions.id', :assessment_id).map { |id, assessment_id| [id, assessment_id] }]
                         .transform_values { |assessment_id| { assessment_id: assessment_id, symptoms: {} } }
-      symptoms&.each do |symptom|
-        conditions_hash[symptom[:condition_id]][:symptoms][symptom[:name]] = symptom.value
+
+      # Uses index_symptoms_on_condition_id
+      all_symptoms = Symptom.where(condition_id: conditions_hash.keys)
+      all_symptoms.where(type: 'BoolSymptom').pluck(:condition_id, :name, :bool_value).each do |(condition_id, name, bool_value)|
+        conditions_hash[condition_id][:symptoms][name] = bool_value
+      end
+      all_symptoms.where(type: 'IntegerSymptom').pluck(:condition_id, :name, :bool_value).each do |(condition_id, name, int_value)|
```

Comment:
Should `:bool_value` should be changed to the integer value?

Reply:
good catch, thanks!

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/717/commits/be040f59effcfd74e377c6b4e273bda72a75ba7c#diff-c34521fcd0fdc2034d456886aae8fd35299fe908ab88d8e49c60d0b4d39ab26bR346